tests/Xamarin.Forms-Performance-Integration/Droid/Xamarin.Forms.Performance.Integration.Droid.csproj

```diff
@@ -18,6 +18,7 @@
     <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
     <AndroidUseAapt2 Condition="'$(AndroidUseAapt2)' == ''">True</AndroidUseAapt2>
     <AndroidDexTool Condition=" '$(AndroidDexTool)' == '' ">d8</AndroidDexTool>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
```

Comment:
I think this should stay in the Release configuration?

Reply:
I've never understood why it was only in Release configuration - after all, Debug configuration is tested on various devices and there's no reason not to build the same architectures for all the configurations.

Reply:
Ah, never mind, I misread is as move to Debug only.